lib/chef-dk/policyfile/local_lock_fetcher.rb

```diff
@@ -0,0 +1,71 @@
+require "chef-dk/policyfile_lock"
+
+module ChefDK
+  module Policyfile
+    class LocalLockFetcher
+
+      attr_reader :name
+      attr_reader :source_options
+      attr_reader :storage_config
+
+      def initialize(name, source_options, storage_config)
+        @name = name
+        @source_options = source_options
+        @storage_config = storage_config
+      end
+
+      def valid?
+        errors.empty?
+      end
+
+      def errors
+        error_messages = []
+
+        [:local].each do |key|
```

Comment:
I'd prefer we change this to `:path` and update the RFC. For cookbooks we use `:path`